data.json

```
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T18:44:53.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "CodingBlocks",
+    "difficulty": "medium",
+    "description": "Have one merged PR on any of <a href=\"https://cb.lk/gh\">Coding Blocks open source projects</a> and get t-shirt with stickers or even a bag (1 PR - t-shirt, 5+ PRs - bag)",
```

```suggestion
    "description": "Have one merged PR on any of <a href='https://cb.lk/gh'>Coding Blocks open source projects</a> and get t-shirt with stickers or even a bag for 5+",
```

Not sure about that one, wdyt?